src/utilities/_hypre_utilities.hpp

```diff
@@ -2475,6 +2490,8 @@ template <typename T>
 HYPRE_Int hypreDevice_CsrRowPtrsToIndicesWithRowNum(HYPRE_Int nrows, HYPRE_Int nnz,
                                                     HYPRE_Int *d_row_ptr, T *d_row_num, T *d_row_ind);
 
+template<typename T1, typename T2, typename T3>
+HYPRE_Int hypreDevice_Axpyzn_mp(HYPRE_Int n, T1 *d_x, T2 *d_y, T3 *d_z, T1 a, T2 b);
```

Comment:
Why `hypreDevice_`?  There are other functions that append `Host` and `Device` just after the `Axpy` part (and similar for other functions).  Is this a GPU naming convention that I am just forgetting about?  I see there is also a `hypreFunctor_` function.

Reply:
My understanding and reason was what @victorapm said :-). I also agree it would be good to document this somewhere, so we are all on the same page.